doc: samples: Change from firmware nRF Cloud to AWS S3 Bucket (#1412)
NordicPlayground/fw-nrfconnect-nrf: samples/nrf9160/aws_fota/README.rst

The `AWS IoT Developer Guide`_ contains all required information about the Amazon Web Services IoT service.

.. note::

>    An file server instance that hosts the new firmware image is required for this sample.

Reviewer: Change to "A file server..."

Author: ```suggestion
   A file server instance that hosts the new firmware image is required for this sample.
```

>    You can set up and use your own AWS S3 server bucket to host the firmware described in the `AWS S3 bucket setup`_ section.

Author: Renamed

>    You can set up and use your own AWS S3 server bucket to host the firmware described in the `AWS S3 bucket setup`_ section.

Reviewer 2: No?

* An `AWS account`_ with access to Simple Storage Service (S3) and the IoT Core service

> * Or an `nRF Cloud`_ account

Author: The goal was to be semi compatible with nRF Cloud giving users the possibility of using what they preferred but maybe just remove it.

> * Or an `nRF Cloud`_ account

Reviewer 2: Yes, remove it - we don't have any other mention of nRF Cloud here anymore.

> * Or an `nRF Cloud`_ account

Author: Okei 👍

#. Go to **Shadow** and confirm that the application version (``nrfcloud__dfu_v1__app_v``) is the one that you configured for the sample.
#. In the :ref:`configuring`, change the application version.
   Then rebuild the application, but do not program it.
#. Go to `AWS S3 console`_ and sign in.
#. Go to the bucket you have created.
#. Click **Upload** and select the file ``app_update.bin`` (located in the ``zephyr`` subfolder of your build directory).

> #. Click the file you uploaded in the bucket and find the download URL to the file through the *Object URL* field.

Author: added

   Certificates must be formatted correctly, without extra whitespace.

Content range is not defined:

>    If you host the firmware image on a different server than in an S3 bucket, this error indicates that the Content-Range field is missing in the HTTP GET header.

Author: A public S3 bucket will include the content-range header at least.

>    To fix this problem, configure the host server to provide this field. Also it could be that you have not configured the S3 bucket correctly. Please check that your file is available from the browser withouth being logged in to your AWS account.

Reviewer: Add comma after "Also, it could be..."

withouth --> without

Author: ```suggestion
   To fix this problem, configure the host server to provide this field. Also, it could be that you have not configured the S3 bucket correctly. Please check that your file is available from the browser without being logged in to your AWS account.
```

>    To fix this problem, configure the host server to provide this field. Also, it could be that you have not configured the S3 bucket correctly. Please check that your file is available from the browser without being logged in to your AWS account.

Reviewer 2: ```suggestion
   To fix this problem, configure the host server to provide this field. 
   If you are using an S3 bucket, make sure that you have configured it as described in `Setting up an AWS S3 bucket`_. 
   Also, confirm that your file is available from the browser without being logged into your AWS account.
```

Author: Added
